analytics.tf

```diff
@@ -0,0 +1,100 @@
+resource "aws_s3_bucket" "dynamodb_s3export" {
+  bucket = "${var.name}-dynamodb-exports"
```

Comment:
we can add an expiration here so that all objects are automatically deleted after 7 days

Reply:
Added a `aws_s3_bucket_lifecycle_configuration` resource to automatically expire objects after 7 days 👍